src/control/cmd/dmg/auto_test.go

```diff
@@ -179,6 +179,7 @@ access_points:
 - hostX:10002
 fault_cb: ""
 hyperthreads: false
+enable_vmd: null
```

Comment:
A little confused by this -- shouldn't this be omitted?

Reply:
done